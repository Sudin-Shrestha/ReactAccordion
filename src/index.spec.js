import index from './App';

test('Everthing is working', () => {
    expect(index).toMatchSnapshot();
    
});