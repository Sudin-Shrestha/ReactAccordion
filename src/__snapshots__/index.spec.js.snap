// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Everthing is working 1`] = `[Function]`;
